modules/cesnet/cesnet.py

```diff
@@ -25,7 +25,7 @@ def init(self):
         self.stop_module = False
 
     def read_configuration(self):
-        """Read importing/exporting preferences from slips.conf"""
+        """Read importing/exporting preferences from slips.conf/slips.yaml"""
```

Comment:
we can completely delete slips.conf, once this PR is merged we wont be using it anymore. in case we do neeed it , we cat always get the code back using git, right?